test/integration/targets/cs_instance/tasks/main.yml

```diff
@@ -9,3 +9,12 @@
 - include: present_display_name.yml
 - include: absent_display_name.yml
 - include: cleanup.yml
+
+#- include: setup.yml
+#  tags: test_cs_instance_adv
+#- include: present_adv.yml
+#  tags: test_cs_instance_adv
+#- include: absent.yml
+#  tags: test_cs_instance_adv
+#- include: cleanup.yml
+#  tags: test_cs_instance_adv
```

Comment:
I check again. These tests are related to features I am developing regarding advanced networking. So these tests do not reflect the current state. I'll remove them.